update with new guardian results (#10)
greenelab/nature_news_manuscript: content/04.results.md

In contrast to _Nature_, in the _Springer Nature_ set, we see a difference in predicted European name origins, with a growing over-enrichment.
Additionally, we see a difference in predicted Arabic/Turkish/Persian name origins frequencies between cited authors and _Springer Nature_ authors, however the absolute difference is lower than observed for Celtic/English and East Asian predicted name origins.

We then sought to determine whether or not the quoted speaker demographic replicated the cited authors’ over- and under-enrichment patterns using articles from _The Guardian_ and _Nature_.
We found a much stronger Celtic/English over-enrichment in comparison to citation patterns, with quotes from those with Celtic/English name origins at a much higher frequency than quotes from those with European name origins (Figure {@fig:suppfig3}d).

> Furthermore, we also found that the Celtic/English over-enrichment was much stronger in _The Guardian_ in comparison to _Nature_.

Reviewer: ```suggestion
Furthermore, we also found that the Celtic/English over-enrichment was much stronger in _The Guardian_ than in _Nature_.
```
